install-devservers.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+# Find user name using `whoami`
+# Find home directory using `echo "$HOME"`
+# Change the user name and home directory to yours in the shell script
+export user_name='mingjie'
+export home_dir='/home/mingjie'
+export venv_dir="$home_dir/.encoded-venv"
+export encd_dir="$home_dir/pandiseased"
+
+export es_install_dir='/opt/elasticsearch'
+export es_package_name='elasticsearch-5.4.3'
+
+export pg_bin_dir='/usr/lib/postgresql/11/bin'
+
+msg='Basic system install'
+echo -e "\n\n$msg"
+if [ -d "$venv_dir" ]; then
+    echo -e "\tAlready installed"
+else
+    sudo apt-get update
+    sudo apt-get install curl
+fi
+echo -e "\n"
+echo -e "DONE $msg"
+
+msg='Installing JAVA: openjdk-11-jre-headles'
+echo -e "\n\n$msg"
+if [ -d "$venv_dir" ]; then
+    echo -e "\tAlready installed"
+else
+    sudo apt install -y openjdk-11-jre-headless
+fi
+echo -e "\n"
+java -version
+echo -e "DONE $msg"
+
+msg="Installing Elasticsearch 5.4.3: $es_install_dir"
+echo -e "\n\n$msg"
+if [ -f "$es_install_dir/$es_package_name/bin/elasticsearch" ]; then
+    echo -e "\tAlready installed"
+else
+    sudo mkdir -p $es_install_dir
+    sudo chown "$user_name:$user_name" "$es_install_dir"
+    cd "$es_install_dir"
+    curl -L -O "https://artifacts.elastic.co/downloads/elasticsearch/$es_package_name.tar.gz"
+    tar -xvf "$es_package_name.tar.gz"
+    export PATH=$es_install_dir/$es_package_name/bin:$PATH
+    echo "export PATH=$es_install_dir/$es_package_name/bin:$PATH" >> "$home_dir/.bashrc"
+    source "$home_dir/.bashrc"
+    sudo -u root chown -R mingjie:mingjie "$es_install_dir/$es_package_name"
```

Comment:
I changed this command to `sudo -u root chown -R mingzhu:mingzhu `, run again and still not working.. Not sure this is the reason or not. I need to reinstall to make sure.